packages/client/src/app/services/auth/auth.service.ts

```diff
@@ -108,7 +111,7 @@ export class AuthService {
     // Fire and forget. If this fails, it doesn't matter to the user, 
     // and we don't want to leak that fact anyway.
     this.http.get(`${this.url}/logout`, { withCredentials: true }).subscribe();
-
+    this.cookies.remove('contentFilter');
```

Comment:
nitpick: maybe leave a newline here, to make it clear the above comment only refers to the HTTP logout call